test/Libraries/Microsoft.Extensions.AI.Integration.Tests/ChatClientIntegrationTests.cs

```diff
@@ -942,7 +942,7 @@ public virtual async Task OpenTelemetry_CanEmitTracesAndMetrics()
 
         var activity = Assert.Single(activities);
         Assert.StartsWith("chat", activity.DisplayName);
-        Assert.StartsWith("http", (string)activity.GetTagItem("server.address")!);
+        Assert.Contains(".", (string)activity.GetTagItem("server.address")!);
```

Comment:
`Assert.Contains(".")` seems likely to have false positives, but I'm not sure what other patterns would make sense. There are ample other asserts in this integration test too, so this is probably fine.